### PR TITLE
MM-24812 Add e2e tests for adding reactions to existing reactions in archived channel posts

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Add e2e tests for adding reactions to existing reactions in archived channel posts.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28412

PS: we don't show reactions in search results, so excluding that from test steps

### CODE REVIEW

**Review by calebroseland — APPROVED** [review]
LGTM

**Review by devinbinnie — APPROVED** [review]

File: e2e/cypress/integration/archived_channel/archive_channel_addReaction_spec.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            cy.get(`#postReaction-${postId}-slightly_frowning_face .Reaction__number--display`).should('have.text', '1');
+
+            // # Remove focus so we can open RHS
+            cy.get('#channel_view').click();
+
+            // # Open RHS for the post
+            cy.clickPostCommentIcon(postId);
+
+            cy.get(`#rhsPost_${postId}`).within(() => {
+                // * Should have add reaction button
```
Comment (jgilliam17) [inline e2e/cypress/integration/archived_channel/archive_channel_addReaction_spec.js:70]:
```suggestion
                // * Should not have add reaction button
```

**Review by jgilliam17 — APPROVED** [review]
Thank you @sudheerDev 
Looks good to merge. Requesting just one small typo to be fixed.
- Test case steps, details and links updated.

**Pull request merged**